examples/BresserWeatherSensorMQTTCustom/src/WeatherSensor.cpp

```diff
@@ -675,24 +673,26 @@ DecodeStatus WeatherSensor::decodeBresser6In1Payload(uint8_t *msg, uint8_t msgSi
     sensor[slot].chan        = chan_tmp; 
     
     
-    // temperature, humidity(, uv) - shared with rain counter
-    temp_ok = humidity_ok = (flags == 0);
+    // temperature, humidity, shared with rain counter, only if valid BCD digits
+    temp_ok  = humidity_ok = msg[12] <= 0x99 && (msg[13] & 0xf0) <= 0x90;
```

Comment:
I wonder why the code has been switched back to not using the flags? This proved to be more robust than the previous solution.